src/blocks/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var(--card-background);
+  padding: 0 16px 16px ;
+}
+
+.card__image {
+  margin: 32px 0 40px;
+  width: 160px;
+  height: 134px;
+  align-self: center;
+
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  vertical-align: top;
+  text-align: left;
+  margin: 0 0 4px ;
+  color: var(--card__title-color);
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--card__product-code-color);
+  margin: 0 0 16px ;
+  text-align: left;
+}
+
+.card__stars-and-reviews {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--card__reviews-color);
+  align-self: flex-end;
+}
+
+.card__price-line {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px 0 16px ;
+}
+
+.product-price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--card__product-price-value-color);
+}
+
+.product-price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--card__product-price-color);
+}
+
+.card__buy-reference {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  background-color: var(--card__button-background-color);
+  border: none;
+  border-radius: 5px;
+  color: white;
+  font-weight: 700;
+  font-size: 14px;
+  text-decoration: none;
+
+}
+
+.card__buy-reference:hover {
+  background-color: #fff;
+  border-color: var(--card__button-background-color);
+  border: solid 1px;
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/213157994-c15e1cbb-1409-4f6c-894d-be6133c1886f.png)

Reply:
fix this comment

Reply:
you can use transparent color and change it on hover